keyrings/alt/pyfs.py

```diff
@@ -9,14 +9,44 @@
 from keyring.backend import KeyringBackend, NullCrypter
 from . import keyczar
 
+# support getting module version
+import pkg_resources
+
+# support parsing version string
+from packaging import version
+
+# check FS version
 try:
-    import fs.opener
-    import fs.osfs
-    import fs.errors
-    import fs.path
-    import fs.remote
+    FS_VER = version.parse(pkg_resources.get_distribution("fs").version)
+except pkg_resources.DistributionNotFound:
+    raise ImportError
+
+import fs.osfs
+import fs.errors
+import fs.path
+
+try:
+    if FS_VER < version.parse("2.0.0"):
```

Comment:
I'd like to see this fs compatibility behavior extracted into a module or function.